<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemForm</class>
 <widget class="QWidget" name="itemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>516</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lst_Items">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>ATKv</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="chk_Comsumable">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Comsumable</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>MPv</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="txt_Description"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>DEFv</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="num_ATKv">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_Name"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>HPv</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="num_HPv">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="num_MPv">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="num_DEFv">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Type</string>
   </property>
   <widget class="QRadioButton" name="rad_null">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_Potion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Potion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_TPotion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Potion</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_Head">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Head</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_Shoes">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shoes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_Armor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Armor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Rad_Legs">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Legs</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_Weapon">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weapon</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Add">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Del">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
